openmdao/docs/openmdao_book/features/model_visualization/real_time_optimization_progress_plot.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ee8e8873",
+   "metadata": {},
+   "source": [
+    "# Using the Real-time Optimization Progress Plot\n",
+    "\n",
+    "The real-time optimization progress plot feature allows the user to see, in real-time, the progress of an optimization run by plotting design variables, constraints, and the objective.\n",
+    "\n",
+    "There two different ways that the plot can be generated."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c48021c2",
+   "metadata": {},
+   "source": [
+    "## Running the plot as part of an optimization run\n",
+    "\n",
+    "Likely the most common way users will use this feature is to run the optimization script is using the `opendmao rtplot` subcommand.\n",
+    "\n",
+    "```\n",
+    "openmdao rtplot path_to_script.py\n",
+    "```\n",
+    "\n",
+    "This will run the optimization script and also automatically launch the plotting app in a browser window at the appropriate time during the optimization run. \n",
```

Comment:
It seems to me that, to be consistent, this should be an option rather than a subcommand...  much like the  `openmdao --view_reports script.py`  command...  The `openmdao` command itself will run the script, with the _option_ to get the real time plots.

https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/utils/om.py#L748

Just a thought...

Reply:
I like that idea but it causes a problem in that there are options to the rtplot subcommand so hard to handle those when the subcommand because an option itself.